apps/studio/components/layouts/ProjectSettingsLayout/SettingsMenu.utils.tsx

```diff
@@ -119,8 +126,9 @@ export const generateSettingsMenu = (
               {
                 name: 'Storage',
                 key: 'storage',
-                url: `/project/${ref}/settings/storage`,
+                url: `/project/${ref}/storage/settings`,
                 items: [],
+                rightIcon: <ArrowUpRight strokeWidth={1} className="h-4 w-4" />,
```

Comment:
Storage and Database hang-around here as ‘shortcuts’ in Phase 1. Hence the `ArrowUpRight` icons. They will be removed in [Phase 2](https://github.com/supabase/supabase/tree/feat/consolidate-settings-phase-2).